mappings/net/minecraft/client/toast/ToastManager.mapping

```diff
@@ -6,7 +6,7 @@ CLASS net/minecraft/unmapped/C_eqfjupcc net/minecraft/client/toast/ToastManager
 		ARG 1 client
 	METHOD m_glbfkboq getToast (Ljava/lang/Class;Ljava/lang/Object;)Lnet/minecraft/unmapped/C_zqhkdxaf;
 		ARG 1 toastClass
-		ARG 2 type
+		ARG 2 id
 	METHOD m_lnyjvhnt clear ()V
 	METHOD m_lsmszfie getNotificationDisplayTime ()D
```

Comment:
```suggestion
	METHOD m_lsmszfie getNotificationDisplaySeconds ()D
```
I think either this^ or `SystemToast.Id.displayDuration` -> `SystemToast.Id.displayMilis`

Reply:
I actually like your latter suggestion! Lemme see if I can apply it